### ISSUE TITLE
Central water tank storage uses PTES data

### ISSUE THREAD

**Issue by amos-schledorn** [PyPSA/technology-data#143]
## Current behaviour
`central water tank storage` uses pit storage data (key `140 PTES seasonal` in DEA techn. catalogue).
I am finding this slightly confusing, since parameters are quite different between the two technologies (such as orders of magnitude difference in losses and CAPEX).

## Expected behaviour
I would either expect 
- `central water tank storage` to be called `pit thermal energy storage` (or similar) or
- the key to be `141 Large hot water tank`

Is this intended?
Happy to prepare a fix otherwise.

**Comment by lisazeyen** [comment]
@amos-schledorn I don't think that this is intended. Would be great if you could prepare a fix!
